test/blackbox-tests/test-cases/reporting-of-cycles.t/run.t

```diff
@@ -67,22 +67,3 @@ cryptic and can involve unrelated files:
   -> required by _build/default/indirect/a.exe
   -> required by alias indirect/indirect-deps in indirect/dune:6
   [1]
-
```

Comment:
What's the fixed message? It's probably better to keep it rather than remove the test case.

Reply:
This test case no longer results in an error. From reading https://github.com/ocaml/dune/issues/4345 I was under the impression that this case shouldn't result in an error, and the dependency cycle was due to a bug in dune. It now occurs to me that we should keep this test around to detect if this bug comes back, so I've added test/blackbox-tests/test-cases/github4345.t